### ISSUE TITLE
REVO

### ISSUE THREAD

**Issue by Tomcatt** [TauLabs/TauLabs#2243]
Hello,

I see you support the REVO mini, but am I able to use this on the full REVO?

**Comment by buzzdev** [comment]
Hi @Tomcatt i use it on Revo Mini and also on Full Revolution.
It works on both.

**Comment by mlyle** [comment]
@tomcatt / @buzzdev Note that Tau Labs doesn't seem to be getting updated anymore, and we've done significant things to improve Revolution support in dRonin ( http://dronin.org )

**Comment by Tomcatt** [comment]
Woo, bro something new? I'm not home right now to download it. Let me see if I can download it.

Okay if TauLabs is dead, LiberPilot still gets updates here and there and not really wild about PID tuning on RaceFlight so I came back. 

I like testing new things. I'll give it a try.

**Comment by buzzdev** [comment]
@mlyle Yes, i know. In fact i am using dRonin on both FCs, while still tempting RaceFlight.
His question was, if TauLabs runs on REVOs, which it does.

It seems, that it is still "being worked" on TauLabs though:
`peabody124 committed on Apr 30`

**Comment by Tomcatt** [comment]
Do you guys have really good community here?

**Comment by mlyle** [comment]
I think so.  http://forum.dronin.org

**Comment by buzzdev** [comment]
...signed up... 👍

**Comment by Tomcatt** [comment]
Signed up as well
